src/funcs.c

```diff
@@ -1909,12 +1909,12 @@ void            ExecEnd (
 *F  FuncSetRecursionTrapInterval( <self>, <interval> )
 **
 */
-
+#include <stdio.h>
```

Comment:
That's why we do code review, no need to say "sorry" (I did far worse things in the past ... let's not talk about it ;-)